### ISSUE TITLE
Please give resolution to the FontStruct-submitted additions!

### ISSUE THREAD

**Issue by apollolux** [google/fonts#2940]
Hello, maker of FontStructions "Trigger" ( #1498 ) and "Trigger Bold" ( #1494 ) here!

It's been a few years since the first set of Google Font addition requests from the previous GF-sponsored FontStruct integrations of submission to GF inclusion were made. For those that were scheduled for eventual inclusion (which does include the two above weights of Trigger), years have passed with no resolution while many other non-FontStruct fonts have since been included. I know at least when I previously requested an update on inclusion progress a reason given for the delays was the focus on increasing variable font support and inclusion, but beyond that years ago no new information and no new actions.

As of Dec 30 2020 I'm seeing 36 FontStruction submissions still open (filter = "is:issue is:open fontstruct") and all but two of them from 2018, with four of them (Trigger Bold, Trigger, Insular Typewriter ( #1526 ), and Blown Out ( #1528 ) ) in the "Add New Families" milestone. Please give resolution to these 36 submissions!

**Comment by apollolux** [comment]
Hello. It's now February of 2023. What updates can you give regarding the status of the FontStruct submissions? Do y'all need the assistance of @robmeek or something? If four submissions originally met the criteria, just add them already, it's been years and other fonts have been submitted and added ahead of them for less.

**Comment by robmeek** [comment]
It does seem unfortunate to leave these font submissions open for so long without any feedback whatsoever– whether the fonts are accepted, rejected or whether changes are needed. 

I understand that the FontStruct submissions were made under specific circumstances – quite different to usual submissions - and this process may have brought its own challeges in terms of curation, but I think that the few designers involved deserve resolution.

**Comment by davelab6** [comment]
I agree, and apologize for not paying attention to this. I'll review the designs with the onboarding folks next month.

**Comment by apollolux** [comment]
Hello! Any updates?

**Comment by apollolux** [comment]
It's February 2024, how about some updates?

**Comment by emmamarichal** [comment]
I sorted the issues, and planed to work on the accepted one for this year (provided I have time for it, and there are no blockers)
thank you for your patience!
